Tabyl label (#575)
sfirke/janitor: NEWS.md

* When using `row_to_names()`, when all input values in `row_number` for a column are `NA`, `row_to_names()` creates a column name of `"NA"`, a character, rather than `NA`. If code previously used relied on a column name of `NA`, it will now error. To fix this, rely on a column name of `"NA"`.

> * `tabyl()` now defaults to displaying the label attribute for the column name (@olivroy, #394).

Author: ```suggestion
* `tabyl()` now defaults to displaying the label attribute for the column name (@olivroy, #394). To revert to previous behavior (ignore variable labels, you can use `haven::zap_labels()`, `labelled::remove_labels()` or similar, before piping in `tabyl()`.

```

* When using `row_to_names()`, when all input values in `row_number` for a column are `NA`, `row_to_names()` creates a column name of `"NA"`, a character, rather than `NA`. If code previously used relied on a column name of `NA`, it will now error. To fix this, rely on a column name of `"NA"`.

> * `tabyl()` now defaults to displaying the label attribute for the column name (@olivroy, #394). To revert to previous behavior (ignore variable labels, you can use `haven::zap_labels()`, `labelled::remove_labels()` or similar, before piping in `tabyl()`.

Reviewer: ```suggestion
* When `tabyl()` is called on a data.frame containing labels, it now displays the label attribute as the name of the first column in the the resulting `tabyl` object (@olivroy, #394). This may break subsequent code that refers to the output of such a `tabyl` by column name. To maintain the previous behavior of ignoring variable labels, you can remove the labels with a function like `haven::zap_labels()` or `labelled::remove_labels()` before calling `tabyl()`.
```

> 
> ## New features
> 
> * `tabyl()` now defaults to displaying the label attribute for the column name (@olivroy, #394).
> 

Author: ```suggestion

* `tabyl()` now defaults to displaying the label attribute for the column name (@olivroy, #394).

## New features

```
